lib/dovico/model/time_entry.rb

```diff
@@ -35,13 +35,13 @@ def self.parse(hash)
     end
 
     def self.get(id)
-      entry = ApiClient.get("#{URL_PATH}/#{id}")["TimeEntries"].first
```

Comment:
Indeed. I did add a trailing slash at first, because [Dovico Doc API uses that](http://apideveloper.dovico.com/Time+Entries). But it works without it, so let's be REST!

I have fixed the other endpoints too.